tt_metal/hw/inc/ethernet/tunneling.h

```diff
@@ -26,7 +26,11 @@ struct eth_channel_sync_t {
     // First level ack that signals to sender that the payload was received by receiver,
     // indicating that sender can reuse the sender side buffer safely.
     volatile uint32_t receiver_ack;
-    uint32_t reserved_1;
+
+    // Logical channel ID tagged by the sender. Not required when channels
+    // are connected 1:1 (single producer - single consumer)
+    volatile uint32_t src_id;
```

Comment:
Can you help to find where instances of `eth_channel_sync_t` are created?
I only see memory being reinterpreted.

Reply:
These are only used in some of the kernels in regions of memory we set aside for synchronizing the channels (they're basically at certain raw addresses in the L1 so they aren't really "created" (though the channel constructor we do clear their contents)